client/src/containers/analysis-visualization/table/mock.tsx

```diff
@@ -0,0 +1,332 @@
+const DATA = [
```

Comment:
You couldn't know how the data will be, It will be more like:

```
const DATA = [
  {
    id: 'a1',
    indicator: 'Carbon emissions',
    values: [
      { year: 2021, value: 200, calculated: false },
      { year: 2022, value: 300, calculated: true } // data projected for non-existing years
    ],
    children: [
      { name: 'Material name', values: [{ year: 2021, value: 100  }]  }
    ]
  }
];
```

For the children, given the indicator id, we will request the values inside in a separate request. For now, I'm happy to include in an object `children` as you are already doing it.